ipe_utils/df_utils.py

```diff
@@ -1,4 +1,6 @@
 import pandas as pd
+from datetime import datetime
+from constants import (WHEN_TO_RUN_SCRIPT, SCRIPT_RUN, COL_COURSE_ID)
```

Comment:
is the () needed in the import statement?

Reply:
That is preferred way for wrapping a long line of imports in Python. This is part of PEP recommendation. https://stackoverflow.com/a/29534096